src/components/Countdown.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+import {
+  intervalToDuration,
+  formatDuration,
+  compareAsc,
+  format,
+} from "date-fns";
+import React, { useState, useEffect } from "react";
+import { pl } from "date-fns/locale";
+
+const eventDate: Date = new Date(2025, 4, 15, 16); //It's the date from pwr calendar, but i have no idea what the actuall hour will be
+
+function Countdown() {
+  const [currentDate, setCurrentDate] = useState<Date>(new Date());
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  useEffect(() => {
+    setCurrentDate(new Date());
+    setIsLoading(false);
```

Comment:
Zeby sie nie odswiezal co sekunde po odliczeniu czasu 
```suggestion
    setIsLoading(false);
    if (compareAsc(eventDate, new Date()) !== 1) return;
```